.github/workflows/symbols-check.yml

```diff
@@ -0,0 +1,41 @@
+name: Symbols Check
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number && format('pr{0}', github.event.number) || github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  Run:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 10
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get build-dep ./
```

Comment:
This is the expensive step :)
```suggestion
```

Reply:
I want the build deps so that 3rd party libraries resolve in clang, right?